tests/e2e/accounts/lockup/continous_lockup_test_suite.go

```diff
@@ -163,17 +164,32 @@ func (s *E2ETestSuite) TestContinuousLockingAccount() {
 		require.NoError(t, err)
 		require.Equal(t, len(ubd.Entries), 1)
 
-		// check if tracking is updated accordingly
-		lockupAccountInfoResponse := s.queryLockupAccInfo(ctx, app, accountAddr)
-		delLocking := lockupAccountInfoResponse.DelegatedLocking
-		require.True(t, delLocking.AmountOf("stake").Equal(math.ZeroInt()))
+		// check if an entry is added
+		unbondingEntriesResponse := s.queryUnbondingEntries(ctx, app, accountAddr)
+		entries := unbondingEntriesResponse.UnbondingEntries
+		require.True(t, entries[0].Amount.Amount.Equal(math.NewInt(100)))
+		require.True(t, entries[0].ValidatorAddress == val.OperatorAddress)
 	})
 
 	// Update context time to end time
 	ctx = ctx.WithHeaderInfo(header.Info{
 		Time: currentTime.Add(time.Minute),
 	})
 
+	t.Run("ok - execute tracking unbonding entry", func(t *testing.T) {
+		msg := &types.MsgTrackUndelegation{
+			Sender: ownerAddrStr,
+			Id:     0,
+		}
+		err = s.executeTx(ctx, msg, app, accountAddr, accOwner)
+		require.NoError(t, err)
+
+		// check if tracking is updated accordingly
+		lockupAccountInfoResponse := s.queryLockupAccInfo(ctx, app, accountAddr)
+		delLocking := lockupAccountInfoResponse.DelegatedLocking
+		require.True(t, delLocking.AmountOf("stake").Equal(math.ZeroInt()))
+	})
```

Comment:
It would be helpful to cover the unhappy path when there is no unbonding or the unboding is not mature, yet.

Personal preference: this is a big test method already. Would it make sense to have a `LockupE2ETestSuite` checks the common operations and focus on the subtype specifics? I am thinking a bit about the way the storage tests are done for the different DBs

Reply:
like spliting to multiple tests method for each actions right ?